app/requirements.txt

```diff
@@ -8,3 +8,4 @@ python-dotenv==0.20.0
 mongoengine==0.24.1
 pytest==7.1.2
 mongomock==4.0.0
+requests==2.27.1
```

Comment:
why do we need this?

Reply:
I had installed it locally but had forgotten to add this.